bitex/formatters/bitfinex.py

```diff
@@ -14,6 +15,8 @@ class BitfinexFormattedResponse(APIResponse):
 
     def ticker(self):
         """Return namedtuple with given data."""
+        if not self.ok:
+            raise PublicDataFetchError(self.ok,self.status_code,self.reason)
```

Comment:
no need to check this

Reply:
It's last error push, had revert
why it exist here?